packages/core/src/domain/configuration/configuration.ts

```diff
@@ -161,7 +160,7 @@ type GenericBeforeSendCallback = (event: any, context?: any) => unknown
  */
 type ProxyFn = (options: { path: string; parameters: string }) => string
 
-interface ReplicaUserConfiguration {
+export interface ReplicaUserConfiguration {
```

Comment:
looks weird but it's because this some type is inferred as `ReplicaUserConfiguration` in another module (remoteConfiguration.ts) but typescript can't _name_ it.
```
@datadog/browser-rum-core: src/domain/configuration/remoteConfiguration.ts(22,17): error TS4058: Return type of exported function has or is using name 'ReplicaUserConfiguration' from external module "/Users/thomas.lebeau/go/src/github.com/DataDog/browser-sdk/packages/core/cjs/domain/configuration/configuration" but cannot be named.
```